test/spec.index.js

```diff
@@ -250,6 +250,57 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('by default, assumes the field isn\'t required', function () {
```

Comment:
That's the old behaviour. But it seems strange to me to assume that a field is required.

If we keep defaulting to true then this PR is pointless (without extra logic). i.e. We test for the presence of the 'required' validator and set aria-required to true only when it is present. If we default to true there is no point in changing anything.

UNLESS, we default to true, and only set to false when fields[key].validate is set but does not contain the required validator. The would make this even less of a breaking change, but might be quite weird behaviour.